src/symbols/Node/Node.tsx

```diff
@@ -139,10 +147,10 @@ export const Node: FC<Partial<NodeProps>> = ({
   onDrag = () => undefined,
   onDragStart = () => undefined,
   onDragEnd = () => undefined,
-  onClick = () => undefined,
-  onKeyDown = () => undefined,
-  onEnter = () => undefined,
-  onLeave = () => undefined
+  onClick = null,
```

Comment:
We could probably kill these now and use `onClick.?()` operators

Reply:
if I kill these lines - TS starts to scream